packages/frontend/src/components/RichTextEditor/MenuBar.tsx

```diff
@@ -0,0 +1,209 @@
+    onClick: (editor: Editor) => {
+      const previousUrl = editor.getAttributes('link').href
+      const url = window.prompt('URL', previousUrl)
+
+      // cancelled
+      if (url === null) {
+        return
+      }
+
+      // empty
+      if (url === '') {
+        editor.chain().focus().extendMarkRange('link').unsetLink().run()
+
+        return
+      }
+
+      // update link
+      editor
+        .chain()
+        .focus()
+        .extendMarkRange('link')
+        .setLink({ href: url })
+        .run()
+    },
+  },
+  {
+    label: 'Remove link',
+    icon: <RiLinkUnlink />,
+    onClick: (editor: Editor) => editor.chain().focus().unsetLink().run(),
+  },
+  {
+    label: 'divider',
+  },
+  {
+    label: 'Heading 1',
+    icon: <LuHeading1 />,
+    onClick: (editor: Editor) =>
+      editor.chain().focus().toggleHeading({ level: 1 }).run(),
+    isActive: (editor: Editor) => editor.isActive('heading', { level: 1 }),
+  },
+  {
+    label: 'Heading 2',
+    icon: <LuHeading2 />,
+    onClick: (editor: Editor) =>
+      editor.chain().focus().toggleHeading({ level: 2 }).run(),
+    isActive: (editor: Editor) => editor.isActive('heading', { level: 2 }),
+  },
+  {
+    label: 'Heading 3',
+    icon: <LuHeading3 />,
+    onClick: (editor: Editor) =>
+      editor.chain().focus().toggleHeading({ level: 3 }).run(),
+    isActive: (editor: Editor) => editor.isActive('heading', { level: 3 }),
+  },
+  {
+    label: 'Heading 4',
+    icon: <LuHeading4 />,
+    onClick: (editor: Editor) =>
+      editor.chain().focus().toggleHeading({ level: 4 }).run(),
+    isActive: (editor: Editor) => editor.isActive('heading', { level: 4 }),
+  },
+  {
+    label: 'Bullet List',
+    icon: <RiListUnordered />,
+    onClick: (editor: Editor) =>
+      editor.chain().focus().toggleBulletList().run(),
+    isActive: (editor: Editor) => editor.isActive('bulletList'),
+  },
+  {
+    label: 'Ordered List',
+    icon: <RiListOrdered />,
+    onClick: (editor: Editor) =>
+      editor.chain().focus().toggleOrderedList().run(),
+    isActive: (editor: Editor) => editor.isActive('orderedList'),
+  },
+  {
+    label: 'divider',
+  },
+  {
+    label: 'Add Table',
+    icon: <RiTableLine />,
+    onClick: (editor: Editor) =>
+      editor
+        .chain()
+        .focus()
+        .insertTable({ rows: 3, cols: 3, withHeaderRow: false })
+        .run(),
+  },
+  {
+    label: 'Add column',
+    icon: <RiInsertColumnRight />,
+    onClick: (editor: Editor) => editor.chain().focus().addColumnAfter().run(),
+  },
+  {
+    label: 'Remove column',
+    icon: <RiDeleteColumn />,
+    onClick: (editor: Editor) => editor.chain().focus().deleteColumn().run(),
+  },
+  {
+    label: 'Add row',
+    icon: <RiInsertRowBottom />,
+    onClick: (editor: Editor) => editor.chain().focus().addRowAfter().run(),
+  },
+  {
+    label: 'Remove row',
+    icon: <RiDeleteRow />,
+    onClick: (editor: Editor) => editor.chain().focus().deleteRow().run(),
+  },
```

Comment:
nit

```suggestion
  },
  {
    label: 'Delete table',
    icon: <RiDeleteBinLine />,
    onClick: (editor: Editor) => editor.chain().focus().deleteTable().run(),
    isDisabled: (editor: Editor) => !editor.isActive('table'),
  },
```

Wondering if we can also have a delete table button?

Reply:
Actually donnid cos they can just highlight the table and delete, vs for row and col they can't do that